client/me/help/help-contact-form/index.jsx

```diff
@@ -99,6 +114,17 @@ export const HelpContactForm = React.createClass( {
 		}
 	},
 
+	doQandASearch() {
+		const query = this.state.subject + ' ' + this.state.message;
+		wpcom.getQandA( query, ( error, data ) => {
+			if ( null === error ) {
+				this.setState( { qanda: data } );
+			} else {
+				this.setState( { qanda: [] } );
+			}
+		} );
```

Comment:
Ah, sorry, for a moment there I thought you meant some wpcom specific Promise enhancements or behaviour :) Have amended this!